### DISCUSSION TITLE
Add support to schedule a publish job

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#78] [Feature Requests]
### What's your feature/idea?

Today the publish status is set globally on the space or space collection. As soon as new changes from a CR are merged, they go live. It’s not the desired use case always, for example you want to review and accept the CR(s) but hold it off until release date. You may also want to give opportunity to release team to review changes, fetch a summary of change-log, and set a date of publish.  

#1 There should be a way to control when a CR should go live. i.e. schedule a CR publish by desired date and time. This should be possible on a per CR basis, i.e. for every CR created a possibility to select release date / time is available. If not set, default should be immediate publish as today.

#2 There should also be support to get a summary of what new CRs will be included in the publish.

### What's your desired outcome?

Be able to review / accept the changes ahead of time, but schedule them for publish at a time in future.
Provide opportunity to other members / stakeholders to review and get/generate change log info. 

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

### Anything else?

The current way poses a risk where undeveloped / untested features can be published ahead of time.
I think the only way to stop it from happening today is to accept the CRs on the release day.

**Comment by ahcsco** [comment]
Any update on this? It's a much required feature for us to be able to plan releases properly, and a standard feature in other documentation apps.

**Comment by aikoii** [comment]
Hey there! Thank you so much for sharing your thoughts and needs. This is something we definitely would like to explore 🧭. I can't say for sure, though, if or when we will build this feature.
